conda/cli/conda_argparse.py

```diff
@@ -948,6 +948,10 @@ def configure_parser_list(sub_parsers):
 
         conda list
 
+    List all packages in reverse order:
```

Comment:
Double colon is needed for proper Sphinx rendering:

```suggestion
    List all packages in reverse order::
```